### PR TITLE
ci: add tslint

### PR DESCRIPTION
**Pull request by d-yokoi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,14 +22,15 @@
...
     "format:ci": "npm run prettier -- --check",
     "integration": "npm run build && npm run nobuild:integration",
+    "lint": "tslint -p tsconfig.json -c tslint.json",
     "nobuild:coverage-report": "nyc report --reporter=lcov",
     "nobuild:coverage-html": "nyc report --reporter=html",
     "nobuild:coverage": "nyc --check-coverage --branches 90 --functions 90 --lines 90 mocha",
     "nobuild:integration": "mocha --timeout 50000 test/integration/",
     "nobuild:unit": "mocha",
     "prepare": "npm run build",
     "prettier": "prettier ts_src/*.ts ts_src/**/*.ts --ignore-path ./.prettierignore",
```
Comment (junderw) [inline package.json]: add `ts_src/**/**/*.ts` to this so that the templates also are formatted.

Comment (d-yokoi) [reply package.json]: fixed
https://github.com/bitcoinjs/bitcoinjs-lib/pull/1353

**Comment by junderw** [comment]
I fixed the code to match the lint, I made 3 changes to tslint config.

Let me know if you think this is OK, and I will merge it.

Thanks again.

```diff
diff --git a/tslint.json b/tslint.json
index 18bf61d2..9708e8b5 100644
--- a/tslint.json
+++ b/tslint.json
@@ -2,6 +2,7 @@
   "defaultSeverity": "error",
   "extends": ["tslint:recommended"],
   "rules": {
+    "arrow-parens": [true, "ban-single-arg-parens"],
     "curly": false,
     "indent": [
       true,
@@ -12,6 +13,8 @@
     "match-default-export-name": true,
     "max-classes-per-file": [false],
     "member-access": [true, "no-public"],
+    "no-bitwise": false,
+    "no-console": false,
     "no-empty": [true, "allow-empty-catch"],
     "no-implicit-dependencies": true,
     "no-return-await": true,
```

**Comment by d-yokoi** [comment]
@junderw 
Thanks for your help!
Your changes to tslint config look good to me.

**Pull request merged**